spec/features/resource_manager_sorts_guiders_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature 'Resource manager sorts guiders' do
+  scenario 'Resource manager sorts guiders', js: true do
+    given_the_user_is_a_resource_manager do
+      and_there_are_some_guiders
+      when_they_sort_those_guiders
+      then_the_guider_order_is_preserved
+    end
+  end
+
+  def and_there_are_some_guiders
+    @guiders = [
+      create(:guider, name: 'A'),
+      create(:guider, name: 'B'),
+      create(:guider, name: 'C')
+    ]
+  end
+
+  def when_they_sort_those_guiders
+    @page = Pages::SortGuiders.new.tap(&:load)
+    @page.reverse_guiders
+    @page.save.click
+  end
+
+  def then_the_guider_order_is_preserved
+    expect(User.guiders).to eq @guiders.reverse
```

Comment:
We should make this same assertion but based on the ordering of the guiders in the UI instead. Also perhaps saying the order was preserved is incorrect, given they've all just been changed?